catalogue/webapp/pages/work.js

```diff
@@ -50,6 +50,7 @@ export const WorkPage = ({ work }: Props) => {
   };
 
   useEffect(() => {
+    window.dataLayer && window.dataLayer.push({ work: work });
```

Comment:
When do we use `if` statements nowadays? I think I'd probably have written one here, but not sure why.